src/components/viewers/MultiFlatmap.vue

```diff
@@ -188,10 +217,10 @@ export default {
           this.toggleSyncMode();
       }
     },
-    multiFlatmapReady: function (component) {
+    multiFlatmapReady: function () {
       this.$refs.multiflatmap.getCurrentFlatmap().enablePanZoomEvents(true); // Use zoom events for dynamic markers
-      this.updateMarkers(component);
       this.flatmapReady = true;
+      this.flatmapMarkerZoomUpdate(true);
       //component.addPanZoomEvent();
```

Comment:
Not part of this PR, I know, but can't say that I like code that is commented out. It's like at lines 268-271 in fact. After all, that's why we have versioning.